### PR TITLE
Dasd 9334/apprentice portal outer api deployment

### PR DESCRIPTION
**Pull request by Knyatsine**

### CODE REVIEW

File: deployments/apprentice-portal.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      Environment: PP
+      ServiceConnection: SFA-DIG-PreProd-ARM
+      AppServiceName: $(ApprenticePortalOuterApiAppServiceName)
+      DeploymentPackagePath: $(Pipeline.Workspace)/das-apim-endpoints-build/ApimEndpointsArtifacts/SFA.DAS.ApprenticePortal.Api.zip
+      ApiVersionSetName: $(ApprenticePortalOuterApiVersionSetName)
+      ApiPath: $(ApprenticePortalOuterApiPath)
+      ApiBaseUrl: $(ApprenticePortalOuterApiBaseUrl)
+      ProductId: $(ApprenticePortalProductId)
+      ApplicationIdentifierUri: $(ApprenticePortalOuterApiIdentifierUri)
+      AppRoleAssignmentsServiceConnection: das-app-role-assignments-CDS
```
Comment (adam230594) [inline deployments/apprentice-portal.yml:148]: wrong service connection

File: deployments/apprentice-portal.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- name: OuterApiObject
+  value: $(ApprenticePortalOuterApiObject)
+
+resources:
+  repositories:
+  - repository: self
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.4.26
```
Comment (adam230594) [inline deployments/apprentice-portal.yml:16]: later versions available

File: deployments/apprentice-portal.yml (hunk 1 above)
Comment (cpwevans) [inline deployments/apprentice-portal.yml]: This is still using CDS service connection. Needs to be corrected.

**Review by cpwevans — COMMENTED** [review]
Happy to approve from a DevOps perspective - encompassing the deployment/build YAML only.

**Comment by Chazzysb** [comment]
APIM code changes approved by commitments team.

**Review by cpwevans — APPROVED** [review]

**Pull request merged**